applications/crossbar/priv/couchdb/account/devices.json

```diff
@@ -13,6 +13,9 @@
         },
         "listing_by_macaddress": {
             "map": "function (doc) {if (doc.pvt_type != 'device' || doc.pvt_deleted) return; if(doc.mac_address) {emit(doc.mac_address, null)}}"
+        },
+        "listing_by_presence": {
+            "map": "function(doc) {if (doc.pvt_type != 'device' || doc.pvt_deleted) return;if (doc.provision && doc.provision.feature_keys) {for(var i in doc.provision.feature_keys) {var key = doc.provision.feature_keys[i];if(key.type == 'presence' || key.type == 'personal_parking') {emit(key.value, key.type);}}}}"
```

Comment:
You don't use the "value" (`key.type`) below. Any reason not to just do `emit(key.value, null)` ? Would save space on the DB servers.